sdk/storage/azblob/service/client.go

```diff
@@ -281,3 +281,40 @@ func (s *Client) FilterBlobs(ctx context.Context, where string, o *FilterBlobsOp
 	resp, err := s.generated().FilterBlobs(ctx, where, serviceFilterBlobsOptions)
 	return resp, err
 }
+
+// NewBatchBuilder creates an instance of BatchBuilder with the specified values.
+//   - cred - an Azure AD credential, typically obtained via the azidentity module
+//   - options - client options; pass nil to accept the default values
+func (s *Client) NewBatchBuilder(cred azcore.TokenCredential, options *ClientOptions) *BatchBuilder {
```

Comment:
Similar to the other constructor, add docs about the behavior.

Reply:
Added.